test/minimal_support/bootsnap_setup.rb

```diff
@@ -0,0 +1,2 @@
+require 'bundler/setup'
+require 'bootsnap/setup'
```

Comment:
Would `Bootsnap.setup` also work here?
I think that should be tested too, as it's used, e.g., in Discourse:
https://github.com/discourse/discourse/blob/3c9c95ac837b1222c2dce43294dd69912a4f43f0/config/boot.rb#L20-L29

Reply:
Yes, ultimately `bootsnap/setup` simply call `Bootsnap.setup` with default options.

Reply:
I see, thanks for the pointer, looks good.